components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/WSDL11SOAPOperationExtractor.java

```diff
@@ -264,7 +280,7 @@ private WSDLSOAPOperation getSOAPOperation(BindingOperation bindingOperation) th
      * @param params      reference parameter list to populate from the parameter nodes
      * @throws APIMgtWSDLException
      */
```

Comment:
Method comment should be updated with the introduced parameter.

Reply:
fixed in 70f09e4